teps/0091-trusted-resources.md

```diff
@@ -292,15 +288,16 @@ type VerificationPolicy struct {
 }
 
 type VerificationPolicySpec struct {
-	// Resources defines the patterns of Resources names that should be subject to this policy. For example, we may want to apply this Policy only from a certain github repo. Then the ResourcesPattern should include the path. If using gitresolver, and we want to config keys from a certain git repo. `ResourcesPattern` can be `https://github.com/tektoncd/catalog.git`, we will use regex to filter out those resources.
-	Resources []ResourcePattern `json:"resources"`
-	// Authorities defines the rules for validating signatures.
-	Authorities []Authority `json:"authorities"`
+	// ResourceAuthorities defines the list of ResourceAuthority, where each contains resource pattern and
+	// corresponding keys
+	ResourceAuthorities []ResourceAuthority `json:"resourceAuthorities"`
```

Comment:
It looks like you have an example yaml with a VerificationPolicy which is still using the previous schema, could that be updated as well? (looking at https://github.com/tektoncd/pipeline/pull/5714 I noticed the doc example there doesnt match the example in the tep here)

Reply:
Oh thanks! updated the examples as well!